platform/mellanox/mlnx-platform-api/sonic_platform/device_data.py

```diff
@@ -153,6 +153,8 @@
         'sfp': {
             'max_port_per_line_card': 16
         }
+    },
+    'x86_64-nvidia_sn5600-r0': {
     }
```

Comment:
@raphaelt-nvidia Why this is empty ?

Reply:
Because the information is not available yet.